src/main/scala/com/microsoft/hyperspace/telemetry/HyperspaceEvent.scala

```diff
@@ -116,6 +116,17 @@ case class CancelActionEvent(appInfo: AppInfo, index: IndexLogEntry, message: St
 case class RefreshDeleteActionEvent(appInfo: AppInfo, index: IndexLogEntry, message: String)
     extends HyperspaceIndexCRUDEvent
 
+/**
+ * Index Refresh Event for appended source files. Emitted when refresh is called on an index
+ * with config flag set to remove index entries for appended source data files.
```

Comment:
Is this a valid comment - `remove index entries`?

Reply:
thanks @imback82 , added

Reply:
fixed teh comment